Tests/conf.json

```diff
@@ -2825,12 +2837,10 @@
         "IsItPhishing": "No instance",
         "Verodin": "No instance",
         "EasyVista": "No instance",
-        "Pipl": "No instance",
         "Moloch": "No instance",
         "Twilio": "No instance",
         "Zendesk": "No instance",
         "GuardiCore": "No instance",
-        "Nessus": "No instance",
```

Comment:
are you synced with @ShahafBenYakir on this ?

Reply:
Reverted